### ISSUE TITLE
Addition of more multi-agent environments

### ISSUE THREAD

**Issue by parthkhopkar** [ArnaudFickinger/gym-multigrid#1]
Hey! Thank you for making this extension to minigrid. Do you plan to add more multi-agent environments in the future such as pursuit evasion, warehouse environment or more environments from literature?

**Comment by ArnaudFickinger** [comment]
Yes I plan to do that! What paper with interesting environments do you have in mind?

**Comment by ayu1812** [comment]
How can I gray highlight my agent's view like it is done in minigrid? Any ideas ?
